src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerReconfigureTaskConsumerTest.java

```diff
@@ -29,8 +29,8 @@ public class TaskManagerReconfigureTaskConsumerTest extends SpringBootContractBa
     private static final String WA_RECONFIGURE_TASK_BY_ID = WA_URL + "/operation";
 
     @Test
-    @PactTestFor(pactMethod = "markTaskToReconfigureById204", pactVersion = PactSpecVersion.V3)
-    void testMarkTaskToReconfigureById204Test(MockServer mockServer) {
+    @PactTestFor(pactMethod = "markTaskToReconfigureById200", pactVersion = PactSpecVersion.V3)
+    void testMarkTaskToReconfigureById200Test(MockServer mockServer) {
```

Comment:
Pact test is broken. We have updated response status to 200 from 204 but actual endpoint n controller is still 204.

Reply:
Looks like it has been changed as part of this PR to OK to comply with the API docs: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1552143414#WATaskManagement:POST/task/operation(internal)-SuccessfulResponses